From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Aditya Prayoga <aprayoga@users.noreply.github.com>
Date: Fri, 22 Jan 2021 13:12:09 +0100
Subject: [ARCHEOLOGY] Fix 2.5G Ethernet on Helios64 Mainline kernel (#2567)

> X-Git-Archeology: > recovered message: > * rockchip64: Added XHCI HCD USB TRB ENT quirk
> X-Git-Archeology: > recovered message: > On some xHCI controllers (e.g. Rockchip RK3399/RK3328/RK1808), they need
> X-Git-Archeology: > recovered message: > to enable the ENT flag in the TRB data structure to force xHC to
> X-Git-Archeology: > recovered message: > prefetch the next TRB of a TD.
> X-Git-Archeology: > recovered message: > Enable the quirk on RK3399 through device tree node properties.
> X-Git-Archeology: > recovered message: > Ported from Rockchip Linux 4.19
> X-Git-Archeology: > recovered message: > * add to dev branch
> X-Git-Archeology: - Revision 568d8472b3c2ab406fd3897e6b952c7a71a8165b: https://github.com/armbian/build/commit/568d8472b3c2ab406fd3897e6b952c7a71a8165b
> X-Git-Archeology:   Date: Fri, 22 Jan 2021 13:12:09 +0100
> X-Git-Archeology:   From: Aditya Prayoga <aprayoga@users.noreply.github.com>
> X-Git-Archeology:   Subject: Fix 2.5G Ethernet on Helios64 Mainline kernel (#2567)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0cdffb29b07305209efb12cf3b5ac6032d3a1153: https://github.com/armbian/build/commit/0cdffb29b07305209efb12cf3b5ac6032d3a1153
> X-Git-Archeology:   Date: Wed, 24 Mar 2021 19:01:53 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Renaming DEV branch to EDGE (#2704)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 3b3d85e25c2ecde30df7b5274fc6f1b9c0299ea2: https://github.com/armbian/build/commit/3b3d85e25c2ecde30df7b5274fc6f1b9c0299ea2
> X-Git-Archeology:   Date: Sat, 22 May 2021 17:08:44 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Upgrade EDGE to 5.12.y (#2825)
> X-Git-Archeology:
> X-Git-Archeology: - Revision e7377248b3cae186e24e2be781cd3365b43246f0: https://github.com/armbian/build/commit/e7377248b3cae186e24e2be781cd3365b43246f0
> X-Git-Archeology:   Date: Thu, 22 Jul 2021 00:15:54 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Second part of EDGE bumping to 5.13.y (#3045)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 6b490e16944b30ff69bf9c13678905187df0d9d4: https://github.com/armbian/build/commit/6b490e16944b30ff69bf9c13678905187df0d9d4
> X-Git-Archeology:   Date: Tue, 11 Jan 2022 15:26:11 +0100
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: move kernel edge to 5.16 (#3387)
> X-Git-Archeology:
> X-Git-Archeology: - Revision f52a4193d02ef88333ba117c68d49486dfd7ff41: https://github.com/armbian/build/commit/f52a4193d02ef88333ba117c68d49486dfd7ff41
> X-Git-Archeology:   Date: Sun, 20 Mar 2022 22:58:21 +0100
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: Adding Pine64 Quartz64a as WIP target (#3539)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0afe24c95729044910e0b3f84dc5500bcdc6524c: https://github.com/armbian/build/commit/0afe24c95729044910e0b3f84dc5500bcdc6524c
> X-Git-Archeology:   Date: Sun, 24 Apr 2022 22:33:47 +0200
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: move kernel media edge to 5.17 (#3704)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 2e1b4eed739d6ea81c3a7ba7e94d1bcab5cbbb6c: https://github.com/armbian/build/commit/2e1b4eed739d6ea81c3a7ba7e94d1bcab5cbbb6c
> X-Git-Archeology:   Date: Mon, 04 Jul 2022 16:18:31 +0300
> X-Git-Archeology:   From: balbes150 <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: move kernel media to current 5.18 and edge 5.19
> X-Git-Archeology:
> X-Git-Archeology: - Revision 738c1613073bfd504aebc223a0d110cc6afefbe5: https://github.com/armbian/build/commit/738c1613073bfd504aebc223a0d110cc6afefbe5
> X-Git-Archeology:   Date: Sat, 12 Nov 2022 10:11:22 +0100
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: move kernel media-current to 5.19.17 (#4406)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 276f346ee9c21cdcd627e54676334f4d5a9292b6: https://github.com/armbian/build/commit/276f346ee9c21cdcd627e54676334f4d5a9292b6
> X-Git-Archeology:   Date: Wed, 14 Dec 2022 10:17:09 +0100
> X-Git-Archeology:   From: Oleg <balbes-150@yandex.ru>
> X-Git-Archeology:   Subject: move kernel media to current 6.0 and edge 6.1 (#4566)
> X-Git-Archeology:
---
 arch/arm64/boot/dts/rockchip/rk3399.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index 873e695b4a8c..d4a65294ba2a 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -461,6 +461,7 @@ usbdrd_dwc3_0: usb@fe800000 {
 			snps,dis_u2_susphy_quirk;
 			snps,dis-del-phy-power-chg-quirk;
 			snps,dis-tx-ipgap-linecheck-quirk;
+			snps,xhci-trb-ent-quirk;
 			power-domains = <&power RK3399_PD_USB3>;
 			status = "disabled";
 		};
@@ -497,6 +498,7 @@ usbdrd_dwc3_1: usb@fe900000 {
 			snps,dis_u2_susphy_quirk;
 			snps,dis-del-phy-power-chg-quirk;
 			snps,dis-tx-ipgap-linecheck-quirk;
+			snps,xhci-trb-ent-quirk;
 			power-domains = <&power RK3399_PD_USB3>;
 			status = "disabled";
 		};
-- 
Armbian

